src/main/java/org/eclipse/yasson/internal/model/JsonbAnnotatedElement.java

```diff
@@ -10,18 +10,27 @@
  * Contributors:
  * Roman Grigoriadi
  ******************************************************************************/
-
 package org.eclipse.yasson.internal.model;
 
+import java.lang.annotation.Annotation;
 import java.lang.reflect.AnnotatedElement;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.json.bind.JsonbException;
+
+import org.eclipse.yasson.internal.properties.MessageKeys;
+import org.eclipse.yasson.internal.properties.Messages;
 
 /**
  * Annotation holder for fields, getters and setters.
```

Comment:
Does this javadoc need to be updated to indicate that it also stores information about classes and interfaces too?

Reply:
Updated it to include classes and interfaces